### PR TITLE
adding unittest for microbenchmarking

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (`hydra unit-tests`)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@aleksbykov can we title this PRs/commit as microbenchmark ? mbm is a kind of cryptic

**Comment by aleksbykov** [comment]
@fruch Of course. I used 'mbm' because it figures in emails.

File: unit_tests/test_mbm.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+import os.path
+import subprocess
+import logging
+import pickle
+import tempfile
+
+from sdcm.microbenchmarking import MicroBenchmarkingResultsAnalyzer, LargeNumberOfDatasetsException, EmptyResultFolder
+
+
+logger = logging.getLogger("microbenchmarking")
```
[the remark this answers is not in the document]
Comment (aleksbykov) [reply unit_tests/test_mbm.py]: fixed

File: unit_tests/test_mbm.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+class TestMBM(unittest.TestCase):
+    def setUp(self):
+        self.mbra = MicroBenchmarkingResultsAnalyzer(email_recipients=('alex.bykov@scylladb.com', ))
+        self.mbra.hostname = 'godzilla.cloudius-systems.com'
+        self.cwd = '/sct/sdcm'
+        self.mbra.test_run_date = "2019-06-27_11:39:40"
+
+    @staticmethod
+    def get_result_obj(path):
```
[the remark this answers is not in the document]
Comment (aleksbykov) [reply unit_tests/test_mbm.py]: The reason was, that in json format dicts failed during comparing, while storing and restoring python object via pickle gives more stable results

**Pull request merged**